config.toml

```diff
@@ -88,12 +90,27 @@ paginate = 6
     # name = "License and Funding"
     # URL = "/privacy"
     # weight = 20
+
+  [[menu.main]]
+  weight = 9
+  name = "Examples"
+  URL = "/examples"
   
 
   [[menu.main]]
-  name = "QCA Infrastructure"
-  URL = "/projects"
+  name = "Infrastructure"
+  URL = "https://docs.qcarchive.molssi.org"
   weight = 10
+  pre = "<u>"
+  post = "</u>"
+
+
+  # footer menu
+  [[menu.footer]]
+  name = "About"
+  URL = "/about"
+  weight = 1
```

Comment:
Did you intend to add the About page back into the footer?

Reply:
I did not, fixed.